node/src/command.rs

```diff
@@ -389,28 +397,21 @@ pub fn run() -> Result<()> {
 
 			// Switch on the concrete benchmark sub-command-
 			match cmd {
-				BenchmarkCmd::Pallet(cmd) =>
+				BenchmarkCmd::Pallet(cmd) => {
 					if cfg!(feature = "runtime-benchmarks") {
 						if !runner.config().chain_spec.is_dev() {
-							return Err("Only dev chain should be used in benchmark".into())
+							return Err("Only dev chain should be used in benchmark".into());
 						}
 
-						runner.sync_run(|config| {
-							if config.chain_spec.is_litmus() {
-								cmd.run::<Block, LitmusParachainRuntimeExecutor>(config)
-							} else if config.chain_spec.is_litentry() {
-								cmd.run::<Block, LitentryParachainRuntimeExecutor>(config)
-							} else if config.chain_spec.is_rococo() {
-								cmd.run::<Block, RococoParachainRuntimeExecutor>(config)
-							} else {
-								Err(UNSUPPORTED_CHAIN_MESSAGE.into())
-							}
-						})
+						use crate::service::HostFunctions;
```

Comment:
Why using HostFunctions instead of old RuntimeExecutor? 
It should not cause too much difference, but what if two parachain's spec_version are different.

Reply:
It's for benchmarking command only - the interface (trait bound) has changed. With the old Executor you'll get:
```
^^^^^ the trait `sc_executor::sp_wasm_interface::HostFunctions` is not implemented for `service::LitmusParachainRuntimeExecutor`
```